venter/tests.py

```diff
@@ -246,27 +250,25 @@ def test_send_mass_mail(self):
         request = SimpleNamespace()
 
         auth_mail = Authorities.objects.create(email='receiver1@example.com', name='receiver')
-
         # Reported Complaint
-        complaints = Complaints.objects.create(created_by=self.user.profile, status=STATUS_REPORTED,
-                                               description='Test Complaint', authority_email=auth_mail)
+        complaint_1 = Complaints.objects.create(created_by=self.user.profile, status=STATUS_REPORTED,
+                                                description='Test Complaint')
+        complaint_1.authorities.add(auth_mail)
 
         # In Progress Complaint with images
-        Complaints.objects.create(created_by=self.user.profile, status=STATUS_IN_PROGRESS, authority_email=auth_mail)
+        complaint_2 = Complaints.objects.create(created_by=self.user.profile, status=STATUS_IN_PROGRESS)
+        complaint_2.authorities.add(auth_mail)
         image = []
-        image.append(ComplaintMedia.objects.create(image_url='https://www.google.com/', complaint=complaints))
-        complaints.images.set(image)
+        image.append(ComplaintMedia.objects.create(image_url='https://www.google.com/', complaint=complaint_1))
+        complaint_2.images.set(image)
 
-        # Complaint with no authority
-        Complaints.objects.create(created_by=self.user.profile, status=STATUS_REPORTED)
-
-        # Test if the email shows up in outbox
+        # Test if the email shows up in the outbox
         queryset = Complaints.objects.filter(status=STATUS_REPORTED)
         complaint_admin.send_emails(complaint_admin, request, queryset)
+        self.assertEqual(list(queryset)[0].email_status, True)
         self.assertEqual(len(mail.outbox), 1)
 
         queryset = Complaints.objects.filter(status=STATUS_IN_PROGRESS)
         complaint_admin.send_emails(complaint_admin, request, queryset)
+        self.assertEqual(list(queryset)[0].email_status, True)
         self.assertEqual(len(mail.outbox), 2)
-
-        self.assertEqual(mail.outbox[0].to[0], auth_mail.email)
```

Comment:
Again, why remove this test?

Reply:
This has been modified to check whether the number of recipients in the 'to' list is as expected.

Reply:
That's fine, but it isn't the same thing. Suppose something breaks such that the authority's names (instead of email) ends up in the to field, this test will still pass, but nothing will work and be hard to detect, especially since we don't really have any logging for this whole thing.

Reply:
Nice